spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -104,21 +104,19 @@ a| * `method` - the fully qualified method name
 
 === Metrics added for `@CircuitBreaker`
 
-ft.circuitbreaker.calls.total{method=<name>, circuitBreakerResult=[success|failure|circuitBreakerOpen]}
```

Comment:
Why did you delete this line?

Reply:
See how it looks in the rendered form. This line clearly isn't supposed to be there.

Reply:
Yeah, suspect I copied and pasted directly from the design in the issue as a placeholder before translating it to have the right formatting for the spec. In this case I forgot to take out the original placeholder.